paasta_tools/cli/cmds/spark_run.py

```diff
@@ -265,6 +249,11 @@ def add_subparser(subparsers):
         default=default_spark_pool,
     )
 
+    list_parser.add_argument(
+        "--aws-instance-types",
+        help="AWS instance types for executor, seperate by comma(,)",
```

Comment:
small wording edit:
```suggestion
        help="AWS instance types for executor, separated by commas (,)",
```

it might also be nice to have arparse handle the splitting for us with something like:
```suggestion
        help="AWS instance types for executor, separated by commas (,)",
        type=lambda instances: [instance for instance in instances.split(","))
```